src/main/scala/com/microsoft/hyperspace/actions/RefreshIncrementalAction.scala

```diff
@@ -49,6 +51,9 @@ class RefreshIncrementalAction(
     logManager: IndexLogManager,
     dataManager: IndexDataManager)
     extends RefreshActionBase(spark, logManager, dataManager) {
+
+  override val fileIdTracker = previousIndexLogEntry.fileIdTracker
```

Comment:
I think it's better to move this fileIdTracker to RefreshActionBase and use it in currentFiles instead of previousIndexLogEntry.fileIdTracker - quick refresh will also need it.

Reply:
`RefreshActionBase` is extended by both `RefreshAction` and `RefreshActionIncremental`. However, `RefreshAction` (i.e. full mode) re-creates index from scratch and that is why I left it use (empty) fileIdTracker from `CreateActionBase`, as we dont need the file Ids from current version. (Everyhing will be recreated).
Are you suggesting moving fileIdTracker initialization as `previousIndexLogEntry.fileIdTracker` to `RefreshActionBase` so in all refresh modes, including the full mode, we re-use existing file ids? 
(There is nothing wrong with that, it is just we dont really need it for the full mode and I kept it only for the incremental mode as that is the only mode needing it now. But if quick mode needs it then your suggestion makes sense. Just wanted to double check with you).

Reply:
Actually even with the current version (wout quick mode) your suggestion makes sense - because anyways `currentFiles` is using it.